gui/src/components/Taipy/Slider.spec.tsx

```diff
@@ -82,6 +82,13 @@ describe("Slider Component", () => {
             type: "SEND_UPDATE_ACTION",
         });
     });
+    it("holds a numeric range", async () => {
+        const { getByDisplayValue } = render(<Slider defaultValue={[10,90]} value={undefined as unknown as number[]} />);
```

Comment:
ça ne peut pas arriver ... si ?
defaultValue={"[10,90]"}

Reply:
Well this does work. As in the 'displays the default value' test.

Reply:
but you are right: the backend does provide a string.
Let me fix that.